### PR TITLE
Javascript/javascript1/week3

### PR DESCRIPTION
**Pull request by AnilMula**

### CODE REVIEW

File: javascript/javascript1/week3/Notepad.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+function getNote(id) {
+  // the function is to find notes in the array of objects
+  if (id == null || isNaN(id)) {
+    return "getNote argument is null or not a number";
+  }
+  for (let i = 0; i < notes.length; i++) {
+    if (notes[i].id === id)
+      return "content is '" + notes[i].content + "' for the id " + notes[i].id;
+  }
+  return "content can not found for id " + id;
```
Comment (zuhairtaha) [inline javascript/javascript1/week3/Notepad.js:17]: the name of the function is `getNote` so it should return a note like this
```js
return notes[i];
```
and at line 17 you can log an error `console.error('content cannot found...)` or throw error like 
```js
throw new Error("content can not found for id " + id);
```
which is better

File: javascript/javascript1/week3/Notepad.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    );
+  }
+}
+// unique feature I think would make this app better is
+// to save a finished note(remove content and change priorities)
+function endedNote(id) {
+  // the function takes id a parameter as parameter
+  // it removes content and id from the notes array
+  // and it updates notes array by changing ids for the rest of objects
+  if (id == null || isNaN(id) || id > notes.length) {
```
Comment (zuhairtaha) [inline javascript/javascript1/week3/Notepad.js:36]: I don't recommend using `isNaN` 
remember that it is a good practice to use `===` not `==`

File: javascript/javascript1/week3/SmartPhoneAppUsage.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  let mm = String(today.getMonth() + 1).padStart(2, "0"); //January is 0!
+  let yyyy = today.getFullYear().toString().substr(-2);
+
+  return dd + "/" + mm + "-" + yyyy;
+}
+function addActivity(date, activity, duration) {
+  // the function will add activies object to activities array
+  activities.push({ date, activity, duration });
+}
+function showStatus(activitiesObject) {
```
Comment (zuhairtaha) [inline javascript/javascript1/week3/SmartPhoneAppUsage.js:14]: the parameter name looks like an object but it is array of objects, so I recommend calling this parameter `activities`

File: javascript/javascript1/week3/TVSeriesContemplate.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    minutes: 0,
+  },
+];
+function logOutSeriesText(tvSeriesDurations) {
+  /*   this function takes an object with properties like
+  name of series, how many days per week it is watched and 
+  duration of the series */
+  // it is assumed that average life of a  person is 80 years(42048000 minutes per 80 years)
+  // the function returns the percentage of a persons life been watching series on tv
+  let seriesTimeInMinutesPerWeek;
```
[the remark this answers is not in the document]
Comment (AnilMula) [reply javascript/javascript1/week3/TVSeriesContemplate.js:27]: i implemented this

File: javascript/javascript1/week3/SmartPhoneAppUsage.js (hunk 3 above)
Comment (AnilMula) [reply javascript/javascript1/week3/SmartPhoneAppUsage.js:14]: changed parameter name

Comment (zuhairtaha) [reply javascript/javascript1/week3/SmartPhoneAppUsage.js:14]: I don't see the changes. anyway I'll approve for now.

**Pull request merged**